### ISSUE TITLE
serifenlose Schriftart

### ISSUE THREAD

**Issue by polyfeux** [DirkRusche/LaTeX#1]
In der packages.sty könnte man folgendes an Zeile 40 einfügen und dann bei Bedarf ein- oder auskommentieren, um eine serifenlose Schriftart zu aktivieren:
```
% sans-serif bzw. Helvatica nutzen:
\usepackage{helvet}
\renewcommand{\familydefault}{\sfdefault}
```

**Comment by DirkRusche** [comment]
Was ist in `\sfdefault` hinterlegt? Oder anders gefragt, welche Schriftart ist Standard?

**Comment by polyfeux** [comment]
Das dürfte die Standardschriftart von LaTeX sein. Also eine Serifen-Schriftart im Stil von "Times New Roman".

**Comment by DirkRusche** [comment]
Done with commit 9ab764b8dc1eff437d7d069f89ca9232e39c7f72. :relaxed: 

Added a switch in the config for it.

**Comment by polyfeux** [comment]
Thank you for adding the switch in the config. Unfortunately the page numbers of the content passage are still in a serif font.

**Comment by DirkRusche** [comment]
Are you sure? Looks like, everything is fine.

Here are some screenshots of the page numbers:

*Compiled with `pdflatex` on Linux*

### Serif

![image](https://user-images.githubusercontent.com/9155644/40533490-44e17ce2-6003-11e8-9df9-93b7acb2e079.png)
![image](https://user-images.githubusercontent.com/9155644/40533510-50a66e20-6003-11e8-843f-3137127b1775.png)

### Without Serif

![image](https://user-images.githubusercontent.com/9155644/40533549-6a5c5fd2-6003-11e8-8578-62d7031cf20e.png)
![image](https://user-images.githubusercontent.com/9155644/40533567-73ed0ca4-6003-11e8-8637-cb8f7aa2b17a.png)

**Comment by DirkRusche** [comment]
@polyfeux Have you already looked this up? :relaxed:

**Comment by polyfeux** [comment]
Yeah, it looks good - thanks for asking! :+1:

**Comment by DirkRusche** [comment]
@polyfeux Okay, ticket can be closed?

**Comment by polyfeux** [comment]
Yeah - sorry, forgot to include that into the comment.

**Comment by polyfeux** [comment]
I'm sorry, to comment on this closed issue, but I found for my system (TeXLive + TeXstudio) a solution to display sans-serif page numbers in the content-part and the following pages:
The following code needs to be added into the meta.sty-files in the if-clause for sans-serif (just after the "\renewcommand"):
`\fancyfoot[c]{\sffamily\thepage}`

I'd suggest I'll add this in a fork of your project some times later. ;)

**Comment by DirkRusche** [comment]
So I doesn't work yet? Because you wrote 2 days ago everything looks fine.

**Comment by polyfeux** [comment]
Yeah, that was because I just had the document with your one-page example.tex.
But after I started to fill the literature.tex and the entire document started to grow, I began to see the other page numbers afterwards.

**Comment by DirkRusche** [comment]
As mentioned [here](#issuecomment-391976155), I couldn't reproduce it.

I will look it up again.
